ngraph_bridge/grappler/ngraph_optimizer.cc

```diff
@@ -51,6 +51,7 @@ Status NgraphOptimizer::Init(
     }
   }
   config_backend_name = params.at("ngraph_backend").s();
+  device_id = params.at("device_id").s();
```

Comment:
May be change this to `config_device_id` to make it consistent with `config_backend_name`